doozerlib/dblib.py

```diff
@@ -161,28 +409,19 @@ def __enter__(self):
         return self
 
     def __exit__(self, *args):
-        if not self.dry_run and self.db.client:
-            with self.db.runtime.get_named_semaphore('package::boto3'):
-                try:
-                    attr_payload = []
-                    for k, v in self.attrs.items():
-
-                        if v is None or v is Missing:
-                            v = ''
-
-                        attr_payload.append({
-                            'Name': k,
-                            'Value': str(v),
-                            'Replace': True
-                        })
-                    self.db.client.put_attributes(
-                        DomainName=self.domain,
-                        ItemName=self.name,
-                        Attributes=attr_payload
-                    )
-                except:
-                    traceback.print_exc()
-                    self.runtime.logger.error(f'Unable to write to simpledb: {self.name}={self.attrs}')
+        with self.db.runtime.get_named_semaphore('dblib::mysql'):
+            try:
+                attr_payload = {}
+                for k, v in self.attrs.items():
+
+                    if v is None or v is Missing or v == '':
+                        v = ''
```

Comment:
this means the entry is entirely left out if the value is empty? if that's the intent... setting `v` at all is confusing. maybe:
```suggestion
                    if v is None or v is Missing or v == '':
                        continue
```

Reply:
Done. Yes makes sense.